src/test/java/automationpractice/com/features/WishlistFeature.java

```diff
@@ -0,0 +1,67 @@
+package automationpractice.com.features;
+
+import automationpractice.com.pages.*;
+import automationpractice.com.steps.LoginSteps;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import net.thucydides.core.annotations.Steps;
+
+import static automationpractice.com.domain.ProductData.DEFAULT_PRODUCT;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class WishlistFeature {
+
+    private static String wishlistName = "newList";
+    private HomePage homePage;
+    private WishlistPage wishlistPage;
+    private MyAccountPage myAccountPage;
+    private ProductDetailPage productDetailPage;
+    private HeaderBox headerBox;
+    @Steps
+    private LoginSteps loginSteps;
+
+    @Given("^logged in customer is on wishlist page$")
+    public void loggedInCustomerIsOnWishlistPage() {
+        homePage.open();
+        loginSteps.login();
+        myAccountPage.clickMyWishlist();
+    }
+
+    @When("^he creates new wishlist$")
+    public void heCreatesNewWishlist() {
+        wishlistPage.insertWishlistName(wishlistName);
+        wishlistPage.submitCreatingWishlist();
+    }
```

Comment:
Ten test nie przejdzie, jeśli już bedziesz miał liste o takiej nazwie. Możesz sprawdzić to w given i ja usunąć jeśli istnieje, lub łatwiej, niech wishlist name bedzie randomowo generowaną nazwą